test/unit/dialects/mssql/resource-lock.test.js

```diff
@@ -2,7 +2,11 @@
 
 const ResourceLock = require('../../../../lib/dialects/mssql/resource-lock'),
   Promise = require('../../../../lib/promise'),
-  assert = require('assert');
+  assert = require('assert'),
+  Support = require(__dirname + '/../../support'),
+  dialect = Support.getTestDialect();
+
+if (dialect !== 'mssql') { return; }
```

Comment:
rather than this wrap in `if (dialect === 'mssql') {`, please do the same for other tests